build/source/driver/multi_driver.f90

```diff
@@ -266,7 +288,7 @@ program multi_driver
 ! nGRU-is the total number of GRUs of the simulation domain
 ! nHRU-is the total number of HRUs of the simulation domain
 ! hruCount-is a local variable for the total number of HRUs in a GRU
-call allocate_gru_struc(nGRU,nHRU,err,message); call handle_err(err,message)
+call allocate_gru_struc(nGRU,nHRU,strtHRU,err,message); call handle_err(err,message)
```

Comment:
See Andy's comment above as well (on lines 133 and following in `allocspace.f90`. It'd be good to come up with consensus on how we want to do this, so everyone follows the same model. It first glance I would prefer to keep them separate, but there may be good reasons not to.